src/App/Action/IndexAction.php

```diff
@@ -23,6 +23,19 @@ public function __construct(TemplateRendererInterface $templateRenderer)
 
     public function __invoke(Request $request, Response $response, callable $next = null) : HtmlResponse
     {
-        return new HtmlResponse($this->templateRenderer->render('app::index'));
+        $currentDate = strtotime(date('U'));
```

Comment:
Lots of terrible assumptions and undefined behaviour when doing time maths like this. Use `\DateTimeImmutable` with `\DateTimeZone` (in UTC) wherever possible, and convert to the client timezone for display purposes only.